docs/apm/traces/view-and-investigate-traces.md

```diff
@@ -59,23 +59,49 @@ Queries above 7 days may be slower to load.
 
 The results are not automatically updated. If you want to refresh traces, click the refresh button on the top right corner of the page.<br/>![Refresh.png](/img/traces/Refresh.png)
 
-### Trace Duration Breakdown Chart
+### Trace Query Visualizations
 
-The Trace Duration Breakdown Chart helps you understand average trace durations in every time bucket as well as the amount of time each service contributed to the end-to-end duration. Use this chart to:
+The Trace Query Visualizations allow you to visualize the breakdown of where time was spent in traces for various services in your application. By aggregating traces, you can gain insights into anomalies and unexpected behaviors, leading to faster Time-to-Resolution (TTR) and a higher Return on Investment (ROI). It offers six charts to help you analyze and trace data efficiently. These include:
+* Trace duration
```

Comment:
```suggestion
* Trace duration as timeseries
```

Reply:
@pbrzoska-sumo I initially thought about specifying the chart type individually. But thank you for the suggestions. I've implemented them.